.do/k8s-cluster/Makefile

```diff
@@ -59,6 +59,30 @@ install_nginx:
 	helm upgrade --install nginx-ingress --version ${NGINX_INGRESS_VERSION} --namespace kube-system ingress-nginx/ingress-nginx \
 	--set controller.publishService.enabled=true
 
+.PHONY: install_certmanager
+install_certmanager:
+	helm repo add jetstack https://charts.jetstack.io
+	helm repo update
+	kubectl label namespace kube-system certmanager.k8s.io/disable-validation="true" --overwrite
+	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.crds.yaml
+
+	# Fetch and patch issuer-letsencrypt.yaml Issuer
+	wget https://docs.codacy.com/chart/values-files/issuer-letsencrypt.yaml >/dev/null 2>&1
+	sed -i issuer-letsencrypt.yaml.bak "s/namespace: codacy/namespace: codacy-dev/g;" issuer-letsencrypt.yaml
+	sed -i issuer-letsencrypt.yaml.bak "s#<--- youremail@yourdomain.com --->#reliability@codacy.com#g" issuer-letsencrypt.yaml
+	kubectl apply --namespace codacy-dev -f issuer-letsencrypt.yaml
+	sed -i issuer-letsencrypt.yaml.bak "s/dev/sandbox/g;" issuer-letsencrypt.yaml
+	kubectl apply --namespace codacy-sandbox -f issuer-letsencrypt.yaml
+	sed -i issuer-letsencrypt.yaml.bak "s/sandbox/release/g;" issuer-letsencrypt.yaml
+	kubectl apply --namespace codacy-release -f issuer-letsencrypt.yaml
```

Comment:
I couldn't get a ClusterIssuer working due to some issues with annotations. I vote for merging this first as it is working, and then a second PR will follow to refactor this into a ClusterIssuer.